sources/api/netdog/test_data/net_config/invalid_dhcp6_options.toml

```diff
@@ -0,0 +1,8 @@
+version = 1
+
+[eno1]
+dhcp4 = true
+
+[eno2.dhcp6]
+optional = true
+route-metric = 100
```

Comment:
A comment in each of these saying why they're invalid might be helpful. It's kind of subtle!